docs.bruno/bruno.json

```diff
@@ -0,0 +1,5 @@
+{
+  "version": "1",
+  "name": "Aniview",
+  "type": "collection"
+}
```

Comment:
Consider adding a newline at the end of the file to comply with POSIX standards [medium]